src/clib/lib/enkf/obs_vector.cpp

```diff
@@ -53,6 +54,22 @@ struct obs_vector_struct {
     std::vector<int> step_list;
 };
 
+static int find_within_tolerance(std::vector<time_t> time_map,
```

Comment:
Perhaps change name to `find_within_tolerance` or something like that.

Reply:
Changed the name, sorry for force pushing